src/renderer/components/ft-create-playlist-prompt/ft-create-playlist-prompt.js

```diff
@@ -0,0 +1,85 @@
+import Vue from 'vue'
+import { mapActions } from 'vuex'
+import FtFlexBox from '../ft-flex-box/ft-flex-box.vue'
+import FtPrompt from '../ft-prompt/ft-prompt.vue'
+import FtButton from '../ft-button/ft-button.vue'
+import FtInput from '../ft-input/ft-input.vue'
+import FtPlaylistSelector from '../ft-playlist-selector/ft-playlist-selector.vue'
+import {
+  showToast,
+} from '../../helpers/utils'
+
+export default Vue.extend({
+  name: 'FtCreatePlaylistPrompt',
+  components: {
+    FtFlexBox,
+    FtPrompt,
+    FtButton,
+    FtInput,
+    FtPlaylistSelector
+  },
+  data: function () {
+    return {
+      playlistName: '',
+      lastActiveElement: null,
+    }
+  },
+  computed: {
+    allPlaylists: function () {
+      return this.$store.getters.getAllPlaylists
+    },
+    newPlaylistVideoObject: function () {
+      return this.$store.getters.getNewPlaylistVideoObject
+    },
+  },
+  mounted: function () {
+    this.lastActiveElement = document.activeElement
+
+    this.playlistName = this.newPlaylistVideoObject.title
+    // Faster to input required playlist name
+    this.$refs.playlistNameInput.focus()
+  },
+  destroyed() {
+    if (this.lastActiveElement != null) {
+      this.lastActiveElement.focus()
+    }
+  },
```

Comment:
```suggestion
    this.lastActiveElement?.focus()
  },
```

Reply:
I think you marked this as resolved on accident